data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/bulk/JavaClientAccumulatingCompressedBulkRequestTest.java

```diff
@@ -36,10 +37,12 @@
 class JavaClientAccumulatingCompressedBulkRequestTest {
 
     private BulkRequest.Builder bulkRequestBuilder;
+    private OpenSearchSink sink;
```

Comment:
Again, we can make all these changes unnecessary with the callback approaches recommended above.